src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -435,7 +435,7 @@
     <value>Object to replace with</value>
   </data>
   <data name="ReplacePortDataResultToolTip" xml:space="preserve">
-    <value>If condition(item) = True, then "replaceWith" is returned. Otherwise "item" is returned unaltered.</value>
+    <value>Item, or list with items replaced by condition</value>
```

Comment:
The wording is strange here. The items are not replaced by a condition, but rather they are conditionally replaced.

Reply:
Addressed in a4098fd